tests/e2e/test_building.py

```diff
@@ -154,3 +154,34 @@ def test_building_account_contract(protostar):
 
     dirs = listdir()
     assert "build" in dirs
+
+
+@pytest.mark.usefixtures("init")
+def test_building_project_with_modified_protostar_toml(protostar):
+
+    with open("./protostar.toml", mode="w", encoding="utf-8") as my_contract:
+        my_contract.write(
+            "\n".join(
```

Comment:
Don't have to change this but multi line strings are a thing ;)

Reply:
We using it as a convention, as it's easier to control the intendation

Reply:
Hmm, usually I just seen either usage of `textwrap.dedent` or smth like this:

```python
def foo():
    if something:
        s = """
blah blabh
blah blah
"""

        trailing_whitespace_sensitive1 = """\
blah blabh
blah blah\
"""

        trailing_whitespace_sensitive2 = """
blah blabh
blah blah
""".strip()
```

Black doesn't yell about this.

Reply:
I don't remember the exat reason tbh I can look for dicussion about this later

Reply:
+1 `dedent`